mewarpx/mewarpx/setups_store/diode_setup.py

```diff
@@ -608,6 +610,8 @@ def init_simcontrol(self):
 
     def init_simulation(self):
         logger.info("### Init Simulation Setup ###")
+        if self.LOAD_BALANCE_INTERVALS > 0:
+            mwxrun.simulation.load_balance_intervals = self.LOAD_BALANCE_INTERVALS
```

Comment:
Should we add some kwargs here to allow for adjusting `load_balance_efficiency_ratio_threshold`, `load_balance_with_sfc`, and `load_balance_knapsack_factor`? The documentation for it including default values is on [this page](https://warpx.readthedocs.io/en/latest/usage/parameters.html?#:~:text=algo.load_balance_intervals).

Reply:
I'll leave that to you for a follow-up PR :smile:

Reply:
I added the flag thinking I'll make a circleCI test with load balancing but since all our tests run serially we decided to just rely on the upstream test.